### ISSUE TITLE
Ability to assign a global prefix

### ISSUE THREAD

**Issue by Pyrodash** [L2jLiga/fastify-decorators#285]
Would it be possible to allow adding a prefix in the bootstrapper? The prefix would be applied to all controllers

i.e.
```js
fastify.register(bootstrap, {
    prefix: '/api/v1',
})
```

**Comment by L2jLiga** [comment]
Hi @Pyrodash,

I think it's already possible and this functionality is provided by Fastify itself, if it isn't then please post the error.

**Comment by Pyrodash** [comment]
It doesn't seem to work right now, no errors but the prefix isn't applied
```ts
fastify.register(bootstrap, {
    controllers: [AuthController],
    prefix: '/api/v1',
})
```
AuthController:
```ts
@Controller({ route: '/oauth2' })
export default class AuthController {
    @GET('/')
    index() {
        return 'hello world'
    }
}
```
The route is only accessible from `http://localhost/oauth2/`

I saw https://github.com/fastify/fastify-plugin/issues/78 and it seems the prefix has to be applied manually?

Maybe the prefix can be manually added to the route in: https://github.com/L2jLiga/fastify-decorators/blob/25c3b3fff015de4d15ab408ee55286581985114d/lib/decorators/controller.ts#L46

**Comment by L2jLiga** [comment]
Published [`3.10.0`](https://github.com/L2jLiga/fastify-decorators/releases/tag/v3.10.0) with support for global prefixing

**Comment by Pyrodash** [comment]
Works perfectly, thank you!
